src/sif/autocost.cc

```diff
@@ -483,6 +493,8 @@ Cost AutoCost::TransitionCost(const baldr::DirectedEdge* edge,
   uint32_t idx = pred.opp_local_idx();
   Cost c = base_transition_cost(node, edge, &pred, idx);
   c.secs = OSRMCarTurnDuration(edge, node, pred.opp_local_idx());
+  c.cost += service_penalty_ *
```

Comment:
The reason it is not in the `base_transition_cost` is because bicycle costing already uses such penalty, but in combination with some other multipliers. Also `truck` profile has `low_class_penalty_` which affects transition to residential and service roads

Reply:
I think it would be fine to add to base transition cost and remove from bicyclecost. I added the penalty recently and I don't think it will impact much moving it into transition cost. Not sure about the truck profile - others @gknisely ? are better able to comment on that.